torchtitan/experiments/qwen3/README.md

```diff
@@ -0,0 +1,21 @@
+**The Qwen3 model is still under development.**
+
+
+
+#### Available features
+QWEN3 0.6B Dense model - Other model sizes are also added to the args, but toml file configs need to be added and tested
+FSDP support
+
+#### Download Qwen3 tokenizer
+
+python scripts/download_tokenizer.py --repo_id Qwen/Qwen3-0.6B
```

Comment:
Add 32B model download instruction , or if all the model share the same tokenizer, ad a note to describe that.